docs/docs/data-fetching.md

```diff
@@ -88,8 +89,11 @@ exports.sourceNodes = async ({
 }
 ```
 
+It is important to note that node fields can indeed be arbitrary. The nameWithOwner and url fields from above are examples of this.
```

Comment:
```suggestion
It is important to note that node fields can indeed be arbitrary. The `nameWithOwner` and `url` fields from above are examples of this.
```

Another nit pick, I like seeing code references wrapped in backticks when possible so they render as `url` and `nameWithOwner`, this another thing I can update!